### DISCUSSION TITLE
How to access externally built Prometheus and Grafana components

### DISCUSSION THREAD

**Discussion by dushulin** [milvus-io/milvus#28653] [Q&A and General discussion]
How to access externally built Prometheus and Grafana components
please give a sample how to fix helm yaml

**Comment by LoveEachDay** [comment]
@dushulin helm chart only create `servicemonitor` custom resource. If your kubernetes doesn't support `servicemonitor`, you'd manually add scrape target for your prometheus.

Some tips:
By default, milvus components will export their metrics at `:9091/metrics`
